frontend/src/game/scenes/Collect.ts

```diff
@@ -52,6 +64,25 @@ export class Collect extends Scene {
   }
 
   update(time: number, delta: number) {
+    this.sweeper.setVisible(false);
+
+    this.sweeper.width = this.bitSweeperSize;
```

Comment:
I think this is making things a little slower maybe - we don't need to change the size of the sweeper every update. we can get rid of these two lines